src/Types/MediaTypes.ts

```diff
@@ -4,11 +4,14 @@ export interface SoundEffects {
   endOfCount: string[]; // by level endof count, rep, set
 }
 
+export enum MediaType {
+  video,
+  audio,
+}
+
 export interface Media {
-  narClip: string;
-  narAutio: string;
-  eduClip: string; // clip id, '' if null
-  eduAudio: string;
-  images: string[]; // image ids one per count, frist one used live
-  soundEffects: SoundEffects;
```

Comment:
Why separate narration from educational media? All we need is the media type and the media clip id, and it is up to the trainer what the content will be